spec/e2e/step_definitions/short_form_steps.coffee

```diff
@@ -68,7 +68,7 @@ module.exports = ->
       i--
 
   @When /^I fill out the Name page as "([^"]*)"$/, (fullName) ->
-    Pages.Name.fill({ fullName })
```

Comment:
@chrisdolendo oops! tests are failing because your search and replace got a little too aggressive 😈  the 2 files in `spec/e2e` should still be `Name` and not `Building_Name`

Reply:
😇  oops, thank you tests!